tests/cli_test.py

```diff
@@ -90,7 +97,7 @@ def test_simple(runner):
 
     result = runner.invoke(main_khal)
     assert not result.exception
-    assert result.output == 'No events\n'
+    assert strip_ansi_escapes(result.output) == 'No events\n'
```

Comment:
I mentioned this in the PR. Without this, the output mismatches.

Remove that function call, and run `pytest -vv`. You'll noticed that the compared string on the left includes the escape codes.

Maybe you have some setting on vdirsyncer for it to do this for you? If so, I'd very much prefer to use that rather than this hack.